src/Slices/DesiredStateDetails/UI/Page.test.tsx

```diff
@@ -132,3 +133,67 @@ test("GIVEN DesiredStateDetails page WHEN api returns items THEN shows success r
     expect(results).toHaveNoViolations();
   });
 });
+
+test("GIVEN DesiredStateDetails page WHEN sorting THEN then we  are sent back to the first page", async () => {
```

Comment:
See two first files, I'm not going to repeat it on every single file :)

Reply:
Can you also update all the other titles of your tests like mentioned in the first files?